_overviews/scala3-macros/tutorial/reflection.md

```diff
@@ -75,6 +98,47 @@ def g[T: Type](using Quotes) =
   ...
 ```
 
+## Symbols
```

Comment:
A thing worth checking is whether the flavor of Markdown used for the docs supports [heading IDs](https://www.markdownguide.org/extended-syntax/#heading-ids). In my experience, adding is a good way of future-proofing internal links, in case a title changes.

Reply:
docs.scala-lang uses kramdown. I ran `jekyll serve` and the internal link seems to work